Assets/Scripts/ResourceLogic.cs

```diff
@@ -0,0 +1,155 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEditor.UIElements;
+using UnityEngine;
+
+public class ResourceLogic : MonoBehaviour
+{
+    //values are out of 100
+    private int paperSupply;
+    private int citizenHappiness;
+    private int money;
+    //value is out of 5 with a HIGHER VALUE = SHORTER GAME!!
+    private int lengthOfGame;
+    private Boolean hasSetLength;
```

Comment:
Fun fact: the primitive version of Boolean in C# is bool (you could make this a bool since you don't need the object version)

Reply:
that's a great idea!